### ISSUE TITLE
Missing Issue Location

### ISSUE THREAD

**Issue by bbertucc** [adobe/pdftools-node-sdk-samples#58]
The PDF accessibility check outputs accessibility issues it encounters but fails to identify where those issues occur. Is there any way to add the location of the failure? 

When scanning a document that has lots of pages, it's hard to understand where the failure is.

Attaching example output from the services API. This output was achieved using the [basic demo.](https://developer.adobe.com/document-services/docs/overview/pdf-services-api/howtos/pdf-accessibility-checker-api/)

[accessibilityChecker2025-04-04T23-56-29.json](https://github.com/user-attachments/files/19760763/accessibilityChecker2025-04-04T23-56-29.json)
